data/models/noteModel.js

```diff
@@ -0,0 +1,138 @@
+const db = require('../dbConfig.js');
+
+const get = async (id) => {
+    if (id) {
+        const note = await db('notes').where('id', id);
+        if (note.length) {
+            return note[0];
+        } else {
+            const e = new Error("id does not exist");
```

Comment:
Might be easiest just to do this all on one line 
```suggestion
            throw new Error("id does not exist");
```

Reply:
Then you won't need the next two lines